docs/home.md

```diff
@@ -118,7 +118,7 @@ It is easy to setup an interactive docs for your own repo. You basically need to
 
     Note: you can also link to external markdown files via their absolute URL (e.g. hosted in another repo), you need be aware that: 1) you need to use the `raw` url 2) the interactive feature won't work on that page.
  5. For Github, you also need to create an empty file named `.nojekyll` (to disable the jekyll site rendering feature of Github).
- 6. To see the result locally, you can `cd docs`, start a static server via `python3 -m http.server 8000`, you can then visit `http://localhost:8000` to see your docs.
+ 6. To **see the result locally**, you need a local Python installation, e.g. with Anaconda. Open a terminal in the `docs` subfolder, and start a static server via `python -m http.server 8000`. You can then visit `http://localhost:8000` to see your docs. Please note, that this site will not be updated. If you change something, you have to restart the static server. 
```

Comment:
Could you try to only refresh the page? It appears to work for me without restart the server.

Reply:
works. i had to refresh twice :-)

Reply:
next commit fixes this.